omas/omas_plot.py

```diff
@@ -1112,68 +1121,87 @@ def sum_density_types(specie_index):
-            axes[3,0].axvline(0.8, ls='--', color='k')
-            axes[3,0].axvline(0.2, ls='--', color='k')
+
         # rotation
         from .omas_physics import omas_environment
         with omas_environment(ods, coordsio={'equilibrium.time_slice.0.profiles_1d.psi': prof1d['grid']['psi']}):
             rotation = (equlibrium['profiles_1d']['centroid']['r_max']-equlibrium['profiles_1d']['centroid']['r_min'])/2 \
                      + equlibrium['profiles_1d']['geometric_axis']['r'] * -prof1d['omega0']
-            axes[3,1].plot(rho_core_prof, rotation, ls=linestyle, lw=linewidth, label=ods['summary']['ids_properties']['comment'] + state_label) # m/s
-            axes[3,1].set_ylabel('R*$\Omega_0$ (m/s)', fontsize='small')
-            axes[3,1].axvline(0.8, ls='--', color='k')
-            axes[3,1].axvline(0.2, ls='--', color='k')
-            axes[3,1].axvline(0.8, ls='--', color='k')
-            axes[3,1].axvline(0.2, ls='--', color='k')
+            axes[3,0].plot(rho_core_prof, rotation, ls=linestyle, lw=linewidth, color=color_label_dict.setdefault(plotting_label,'r'), label=ods['summary']['ids_properties']['comment'] + " " + plotting_label)  # m/s
+            axes[3,0].set_ylabel('R*$\Omega_0$ (m/s)', fontsize='small')
+            axes[3,0].axvline(0.8, ls='--', color='k')
+            axes[3,0].axvline(0.2, ls='--', color='k')
+            axes[3,0].axvline(0.8, ls='--', color='k')
+            axes[3,0].axvline(0.2, ls='--', color='k')
+
+        # plot zeff if plot_zeff is passed to plotting function
+        if plot_zeff:
+            axes[4,0].plot(rho_core_prof, prof1d['zeff'], ls=linestyle, lw=linewidth, color=color_label_dict.setdefault(plotting_label,'r'), label=plotting_label)  # keV
+            axes[4,0].set_ylabel('$Z_{eff}$', fontsize='small')
+            axes[4,0].axvline(0.8, ls='--', color='k')
+            axes[4,0].axvline(0.2, ls='--', color='k')
 
     # Fluxes
     if "core_transport" in ods:
         core_transport = ods['core_transport']['model']
         rho_TGLF = core_transport[0]['profiles_1d'][time_index]['grid_d']['rho_tor']
         # Qe
-        axes[0,1].plot(rho_TGLF, core_transport[2]['profiles_1d'][time_index]['electrons']['energy']['flux'], ls='-', marker="o", markersize=6, lw=2, label="total" + state_label)
-        axes[0,1].plot(rho_TGLF, core_transport[3]['profiles_1d'][time_index]['electrons']['energy']['flux'], ls='-', marker="x", markersize=8, lw=2, label="target" + state_label) # W/m^2
-        axes[0,1].plot(rho_core_prof, core_transport[4]['profiles_1d'][time_index]['electrons']['energy']['flux'], ls='--', lw=3, label="power_balance")
-        axes[0,1].set_ylabel('$Q_e$ (W/$m^2$)', fontsize='small')
+        axes[0,1].plot(rho_TGLF, core_transport[2]['profiles_1d'][time_index]['electrons']['energy']['flux'], ls='-', lw=2.5, color=color_label_dict.setdefault(plotting_label, 'r'), label="total" + plotting_label)
```

Comment:
If it has to be generic it does not make too much sense to call it `rho_TGLF`

Reply:
makes sense changed it to rho_transport_model